packages/backend/src/tests/propertyBased/arbitraries/cardinality.ts

```diff
@@ -0,0 +1,109 @@
+import { fc } from '@fast-check/vitest';
+import {
+    AnySchema,
+    getSelectableColumnsFromGenericDbSchema,
+    getWhereableColumnsFromGenericDbSchema,
+    getTableNamesFromGenericDbSchema,
+} from '@synthql/queries';
+
+type Cardinality = 'one' | 'maybe' | 'many';
+
+export function generateQueryArbitrary(
```

Comment:
This needs some documentation

Reply:
Ideally also try to keep one arbitrary per file. This makes the files easier to read and reason about.